assets/prototype/domain/shared/entities/datetimes/predicates/validStatus/index.test.ts

```diff
@@ -0,0 +1,37 @@
+import { omit } from 'ramda';
+
+import validStatus from './index';
+import { nodes as datetimes } from '../../../../../eventEditor/data/queries/datetimes/test/data';
+
+describe('validStatus', () => {
+	it('should return false if datetime has missing status property', () => {
+		datetimes.forEach((datetime) => {
+			const newDatetime: any = omit(['status'], datetime);
```

Comment:
Nice use of `: any` 😄

Reply:
You taught me this :D

Reply:
oh cuz TS flips out when you remove the prop but try to call it a Datetime? makes sense